plugin-volume/alsaengine.cpp

```diff
@@ -140,8 +142,12 @@ void AlsaEngine::discoverDevices()
         if (cardNum < 0)
             break;
 
-        char str[64];
-        sprintf(str, "hw:%i", cardNum);
+        char str[BUFF_SIZE];
+        n = snprintf(str, BUFF_SIZE, "hw:%i", cardNum);
```

Comment:
`BUFF_SIZE - 1` should be passed as max length since 1 byte is required for the terminator.